app/src/main/java/petermcneil/musiclibrary/services/database/BioService.java

```diff
@@ -0,0 +1,91 @@
+package petermcneil.musiclibrary.services.database;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.ResultSetExtractor;
+
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Service;
+import petermcneil.domain.Bio;
+import petermcneil.musiclibrary.services.CRUDService;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Map;
+
+@Service
+public class BioService implements CRUDService<Bio>{
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+    private static final Logger LOG = LoggerFactory.getLogger(BioService.class);
+
+    public BioService(final NamedParameterJdbcTemplate jdbcTemplate){
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public Bio get(Integer bioId){
+        MapSqlParameterSource params = new MapSqlParameterSource();
+        params.addValue("bioId", bioId);
+        return jdbcTemplate.query("SELECT * FROM bio WHERE idbio=:bioId", params, new ResultSetExtractor<Bio>() {
+            @Override
+            public Bio extractData(ResultSet rs) throws SQLException, DataAccessException {
+                Bio.Builder bio = Bio.bioBuilder();
+                while(rs.next()){
```

Comment:
probably change to an if, not a while. If we have more than 1 bio for a given id something has gone wrong, might want to log a warning or throw an exception